aui.crypt/CMakeLists.txt

```diff
@@ -9,7 +9,19 @@ aui_module(aui.crypt EXPORT aui)
 aui_enable_tests(aui.crypt)
 aui_link(aui.crypt PUBLIC aui::core)
 target_include_directories(aui.crypt PRIVATE ${OPENSSL_INCLUDE_DIR})
-aui_link(aui.crypt PRIVATE OpenSSL::SSL OpenSSL::Crypto)
+
+if(MSVC AND AUI_BUILD_FOR STREQUAL "winxp")
+    include(../WinXPuseLTL5.cmake)
+    aui_link(aui.crypt PRIVATE OpenSSL::SSL OpenSSL::Crypto YY_Thunks)
+    if(BUILD_SHARED_LIBS)
+        set_target_properties(aui.crypt PROPERTIES LINK_FLAGS "/SUBSYSTEM:CONSOLE,5.01 /SUBSYSTEM:WINDOWS,5.01 /ENTRY:DllMainCRTStartupForYY_Thunks")
+    else()
+        set_target_properties(aui.crypt PROPERTIES LINK_FLAGS "/SUBSYSTEM:CONSOLE,5.01 /SUBSYSTEM:WINDOWS,5.01")
+    endif()
+else()
+    aui_link(aui.crypt PRIVATE OpenSSL::SSL OpenSSL::Crypto)
+endif()
+
```

Comment:
These things are repetitive among all modules. Can we move this to `aui_module` or `aui_common`?